stack/new_stack.py

```diff
@@ -373,17 +444,24 @@ def _create_hail_service_account(username, hail_auth_token):
     post_resp.raise_for_status()
 
 
+def _get_standard_hail_account_names(dataset):
+    username_suffixes = ['-test', '-standard', '-full']
+    return [dataset + suffix for suffix in username_suffixes]
```

Comment:
```suggestion
    return ['aggregate-billing'] + [dataset + suffix for suffix in username_suffixes]
```

Reply:
This is used to generate the hail accounts, the billing aggregator username is added when setting up the hail batch billing project.